models/query/report-query.js

```diff
@@ -55,6 +55,7 @@ ReportQuery.prototype.toMongooseFilter = function() {
   if (this.keywords)  filter.content = { $regex: this.keywords,  $options: 'i' }
   if (this.tags)      filter.smtcTags = { $all: this.tags }
   if (this.list)      filter["metadata.ct_tag"] = {$in: [this.list] }
+  console.log(filter);
```

Comment:
Please remove, it logs quite frequently 😀

Reply:
Whoops; thought I got all of these.